src/js/controllers/index.js

```diff
@@ -1,7 +1,8 @@
-import angular from 'angular';
-import {appName} from '../utils/constants';
-import TestController from './TestController';
+import {app} from '../bootstrap';
```

Comment:
This reference can be retrieved by calling `angular.module(appName)` and no other arguments, like, `const app = angular.module(appName);`. thats why i didnt export it initially.